### PR TITLE
Drawer screens and Profile Page

### LINKED ISSUE #50: Drawer Screens
- [x]  Developer's Info
- [ ]  Privacy Policy
- [ ]  About AICHE
- [ ]  About AICHE NIT Rourkela

### PR DESCRIPTION
**Pull request by nayakastha**
### Description
Addition of drawer screens as mentioned in Issue #50 

closes #50 

### Build Status
[pr-build-test](https://github.com/dscnitrourkela/project-maple/actions/workflows/pull_request.yml)

### CODE REVIEW

File: aptiche/lib/views/drawer/dev_info.dart [hunk 1]
```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import '../../services/third_party_services.dart';
+import '../../utils/theme.dart';
+import '../../utils/ui_scaling.dart';
+
+const String aboutText =
```
Comment (SriramPatibanda) [inline aptiche/lib/views/drawer/dev_info.dart:7]: Keep this in Strings file

File: aptiche/lib/views/drawer/dev_info.dart [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                ],
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget contactCard(BuildContext context, String name, String contact) {
```
Comment (SriramPatibanda) [inline aptiche/lib/views/drawer/dev_info.dart:117]: Prefer using Classes insead of functions for Widgets.

File: aptiche/lib/views/drawer/privacy_policy.dart [hunk 3]
```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+import '../../utils/theme.dart';
+import '../../utils/ui_scaling.dart';
+
+const String privacy_policy = 'Privacy Policy\n '
```
Comment (SriramPatibanda) [inline aptiche/lib/views/drawer/privacy_policy.dart:7]: Prefer to move the string to the strings file.

**Review by SriramPatibanda — CHANGES_REQUESTED** [review]
Minor changes requested.

File: aptiche/lib/widgets/HomeScreen/contact_card.dart [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  final ThirdPartyServices thirdPartyServices;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: InkWell(
+        onTap: () {
+          if (contact != "") {
+            thirdPartyServices.urlLauncher(contact);
```
Comment (abhibhaw) [inline aptiche/lib/widgets/HomeScreen/contact_card.dart:26]: URL Launcher isn't working

**Review by abhibhaw — CHANGES_REQUESTED** [review]
@nayakastha Will suggest you to just keep the privacy policy page as an external link/webview.

File: aptiche/lib/widgets/HomeScreen/contact_card.dart (hunk 4 above)
Comment (nayakastha) [reply aptiche/lib/widgets/HomeScreen/contact_card.dart:26]: it is working for me. Also separated the privacy policy into a separate file. `assets/markdown/privacy_policy.md`

**Comment by abhibhaw** [comment]
> image image
> Hello @nayakastha , That's a great improvement to the code. Have a pinch of paitence while the reviewer gets impressed by the changes you made. Here are some doggos for company while you are waiting for the merge and marching ahead with your Hacktoberfest Contributions, Check your [Dashboard](https://hacktoberfest.digitalocean.com/profile) for more information on Hacktoberfest. Stay safe 🚀 .

Such an amazing gif

**Comment by nayakastha** [comment]
closes #52

**Comment by nayakastha** [comment]
> @nayakastha Will suggest you to just keep the privacy policy page as an external link/webview.

https://sites.google.com/view/apticheprivacypolicy/home

**Review by SriramPatibanda — APPROVED** [review]

**Pull request merged**